### ISSUE TITLE
FoodTruckIntent: Add some humor to the response

### ISSUE THREAD

**Issue by jmartini** [codeforboston/voiceapp311#325]
**Is your feature request related to a problem? Please describe.**
Add some humor into the food truck response

**Describe the solution you'd like**
User | Alexa, ask Boston Info if there are any food trucks in the Financial District?
-- | --
Alexa | There are at least 3 food trucks, within a mile from that area, open from 11 AM to 3 PM. At Financial District, you will find: Taco Party, Bim Bim Box, and Mediterranean Home Cooking. 
User | Perfect.
Alexa | Save me some leftovers! Kidding, I'm a robot.

**Comment by Joshhw** [comment]
@jmartini I don't think this is something we can/should be doing. This seems like something Amazon would ding us for.

**Comment by Joshhw** [comment]
@david-iterators so after speaking with @jmartini tonight I have a better understanding of what we can do to make this work. You will need to disable the end session and present a follow up question to make the back and forth work for this to pass. Let me know if you have any questions about this.
